subsys/usb/class/Kconfig

```diff
@@ -152,3 +152,19 @@ config USB_DFU_DETACH_TIMEOUT
 	int
 	depends on USB_DFU_CLASS
 	default 1000
+
+config USB_DFU_DEFAULT_POLLTIMEOUT
+	int
+	depends on USB_DFU_CLASS
+	default 256
+	range 0 1000
+	help
+	  Default value for bwPollTimeout (in ms)
+
+config USB_DFU_DNLOAD_POLLTIMEOUT
+	int
```

Comment:
Can we add a prompt so it will be user-configurable in case this is needed to be changed?

Reply:
sorry, the prompt should be there, will fix

Reply:
@Qbicz fixed

Reply:
Great. For me PR is fine